.reuse/dep5

```diff
@@ -0,0 +1,10 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
The commit message says:

> Do reuse init folder with GPLv2+ license
> and custom template for copyright.

But this adds 3 licenses - please update the commit message to describe
why we add each license.

Adding GPLv2+ duplicate the current COPYING file - this file should be removed when this change is done.

Reply:
About the COPYING, I left it in following the discussion with @mz-pdm : `If LICENSES/ is required then OK. Whether to keep COPYING in such a case: It defines the default licence, which may be relevant for files without copyright information. So it's probably better to keep it.`

Although technically we shouldn't have files without copyright information with this approach.

Reply:
We cannot have files without license since the "reuse lint" job will fail. Keeping COPYING file while we have LICENCES directory with 3 license does not make sense.

Reply:
OK. I will remove it then.

As an addition to the discussion, this topic is also mentioned in the `reuse` FAQ: https://reuse.software/faq/#tradition
They argue why they opt for `LICENSES`, how they think it is better, and how projects may still choose to keep `COPYING` in their root folder, even if not necessary.

Reply:
Please open an issue for removing it.

Reply:
We need to change the spec:

    %license COPYING

We need to install the LICENSES directory with all the licenses, since we really have multiple license.